### PR TITLE
feat(core): store Layer instead of layer's id

### PR DESCRIPTION
**Pull request by peppsac**
It's more convenient (no need to use `View.getLayers()` to retrieve the real layer) and won't be costlier than storing strings.

(I have another PR coming that depends on this)

### CODE REVIEW

File: src/Process/ObjectRemovalHelper.js [hunk 1]
```diff
@@ -23,27 +23,27 @@ export default {
     /**
      * Remove obj's children belonging to layerId layer.
```
Comment (zarov) [inline src/Process/ObjectRemovalHelper.js]: Forgot to change some comments in this file

File: src/Provider/TileProvider.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ function executeCommand(command) {
 
     geometry._count++;
     const tile = new TileMesh(geometry, params);
-    tile.layer = layer.id;
+    tile.layer = layer;
```
Comment (zarov) [inline src/Provider/TileProvider.js:85]: I take this one as an example, but you could remove the `layerId` param above.

**Review by zarov — COMMENTED** [review]
A few things forgotten, but LGTM overall.

**Comment by peppsac** [comment]
Thanks for the review @zarov. I fixed your comments and rebased the PR.

**Review by zarov — APPROVED** [review]
Thanks !

**Pull request merged**